cargo-screeps/src/orientation.rs

```diff
@@ -3,21 +3,16 @@ use std::{env, path::PathBuf};
 use failure;
 
 pub fn find_project_root() -> Result<PathBuf, failure::Error> {
-    // TODO: is it right to canonicalize here?
-    let mut current = env::current_dir()?.canonicalize()?;
+    let orig = env::current_dir()?;
+    let mut current = orig.clone();
 
     loop {
-        if current.join("screeps.toml").exists() {
+        if current.join("Cargo.toml").exists() {
             return Ok(current);
         }
-        let is_last = current.pop();
+        let is_last = !current.pop();
         if is_last {
-            bail!(
-                "could not find 'screeps.toml' in {} or parents.\n\
-                 Please create 'screeps.toml' in project root. (template at \
-                 https://github.com/daboross/screeps-in-rust-via-wasm/blob/master/screeps.toml)",
-                current.display()
-            );
+            bail!("could not find crate root {} or parents", orig.display());
```

Comment:
Ideally I'd like to still point people to the default configuration file when they don't have one. Even if where we actually check for the configuration moves around, the error message for when we don't have a screeps.toml should probably still include this?